ktlint-ruleset-experimental/src/test/resources/spec/indent/format-eq-expected.kt.spec

```diff
@@ -5,21 +5,19 @@ class C {
 }
 
 fun f() {
-    val x =
-        "a" +
-            "b2"
+    val x = "a" +
```

Comment:
i don't know if this better...

Reply:
really? it seems better to me. but the previous formatting is still allowed, just not forced.